content/TrueCommand/Administration/Settings/SAMLAD.md

```diff
@@ -0,0 +1,218 @@
+      *IP:PORT* is your TrueCommand system IP address/port number.
+
+      ![TC2.2PowershellInvokeCommand](/images/SAML/TC2.2PowershellInvokeCommand.png "Open Powershell Type Invoke Command")
+
+   b. Edit the certificate. 
+
+      1. Open a File Explorer window and locate the <file>tc.cer</file> file in <file>C:/local data/user/QE</file>.
+
+         ![LocateTcCertificateWithFileExplorer](/images/SAML/LocateTcCertificateWithFileExplorer.png "Locate TC Certificate With File Explorer")
+      
+      2. Select the <file>tc.cer</file> file, right-click, then select **Open with Notepad**.
+
+      3. Delete everything before the certificate open tag `<x509Certificate>` and the open tag. 
+         Delete any URL immediately following the open tag, and everything after the certificate close tag `</x509Certificate>` and the close tag.
+         The only thing that remains is the certificate with no tags at the beginning and end of it.
+         
+         ![OpenTcCertificate](/images/SAML/OpenTcCertificate.png "OpenTcCertificate")
+              
+      4. Type this string at the beginning of the certificate, exactly as `-----BEGIN CERTIFICATE-----` with five dashes before and after the text with no spaces. 
+     
+      5. Type this string at the end of the certificate, exactly as `-----END CERTIFICATE-----` with five dashes before and after the text with no spaces. 
+  
+         ![EditedTcCertificate](/images/SAML/EditedTcCertificate.png "Edited Tc Certificate")
+      
+      6. Click **Save** (or <kbd>Ctrl-S</kbd>) and then close Notepad, and then Powershell.
+
+   d. Select **Browse** in the **Configure Certificate** Relying Party Trust Wizard window. 
+      The **Encryption Certificate** window opens.
+
+      ![WizardConfigureCertificateSelectBrowseCropped](/images/SAML/WizardConfigureCertificateSelectBrowseCropped.png "Wizard Configure Certificate Select Browse")
+   
+   e. Locate the <file>tc.cer</file> file (<file>C:/local disk/users/QE</file> and then select the <file>tc.cer</file>).
+      Click **Open** to view the **Configure Certificate** window and see information about the certificate. 
+      Click **Next** to move on to the **Configure URL** window.
+
+4. Configure the URL. In the **Configure URL** window:
+
+   a. Select **Enable support for the SAML 2.0 WebSSO protocol**. 
+
+      ![WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped](/images/SAML/WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped.png "Wizard Select Enable Support For SAML 2.0 Web SSO Protocol")
+
+   b. Enter the TrueCommand login URL (http://*IP:PORT*/saml/acs) in the **Relying party SAML 2.0 SSO service URL** field. 
+      *IP:PORT* is your TrueCommand system IP and port address.
+
+   c. Click **Next** to continue to the **Configure Identifiers** window.
+
+5. Configure the SAML identifiers. 
+   
+   a. Type **truecommand-sml** into the **Relying party trust identifier** field and click **Add**.
```

Comment:
Type **truecommand-sml** should be
 Type **truecommand-saml**

Reply:
fixed I thought.